__tests__/fixtures/test_file_list_source_file.txt

```diff
@@ -0,0 +1,4 @@
+__tests__/a.test.js
+__tests__/directory/b.test.js
+
```

Comment:
An empty line is left here on purpose (as well as `\r\n` at the end of the file). We ignore blank links when looking for a test file path (See `TestFilesFinder.testFilesFromSourceFile`).